### DISCUSSION TITLE
ChatGPT and Angular17

### DISCUSSION THREAD

**Discussion by varHoudini** [community/community#84988] [Programming Help] [answered]
Does anyone has a better AI for scripting in Angular17 ? ChatGPT's last update was 2021 and Angular once again had to change the whole code syntax..

**Comment by varHoudini** [comment]
Bard maybe?

**Reply by Mahitej28** [reply, marked as answer]
Hello @varHoudini according to me Bard would definitely will be helpful also if you are using vs code editior, I would like to provide a link that contains some famous vscode extensions that can help u generate better AI script.
https://visualstudiomagazine.com/articles/2023/03/08/vs-code-ai-tools.aspx

**Comment by samus-aran** [comment]
Hi @varHoudini , Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we're glad you're here! 

We want to make sure you’re getting the best support you can now that GitHub Universe has ended so I've gone ahead and moved this to the most relevant category for you.

Thank you for helping us maintain a productive and tidy community for all our members.
